pkg/test/framework/components/echo/kube/sidecar.go

```diff
@@ -56,31 +54,9 @@ func newSidecar(pod kubeCore.Pod, cluster cluster.Cluster) (*sidecar, error) {
 		cluster:      cluster,
 	}
 
-	// Extract the node ID from Envoy.
-	if err := sidecar.WaitForConfig(func(cfg *envoyAdmin.ConfigDump) (bool, error) {
```

Comment:
This is only to get node ID, which is only used to get pod name, which we already have. This adds >1s to each run -- this adds up because re-running locally only takes like 3s total so its 33% overhead.